Unit_testing_frameworks/Calculator_Tests/IsNegativeTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using CSharpCalculator;
+
+namespace Calculator_Tests
+{
+    [TestClass]
+    public class IsNegativeTests
+	{
+        [TestMethod]
+        public void IsNegativeValidResult()
+        {
+            // Arrange
+            double param = 20;
+            Calculator calc = new Calculator();
+
+            // Act
+            calc.isNegative(param);
+
+            // Assert
+            bool actual = calc.isNegative(param);
+            Assert.IsFalse(calc.isNegative(param));
```

Comment:
You call calc.isNegative(param); method 3 times. Why?

Reply:
I call the method after //Act because I saw a sample of designing test methods in Mstest, since the methods being checked are simple, I simply called methods in this block so that the act block was not empty. should it be removed? 
and I just forgot to remove bool actual = calc.isNegative(param); ....

Reply:
You can call a method on a Act and write the result to a variable, and then check that variable in Assert. Or you can do as it is right now.